.github/workflows/ci.yaml

```diff
@@ -289,6 +289,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: ory/kratos-selfservice-ui-node
+          ref: jonas-jonas/two-step
```

Comment:
@jonas-jonas revert?

Reply:
```suggestion
```

Reply:
we'll need to merge then https://github.com/ory/kratos-selfservice-ui-node/pull/342

Reply:
done